src/llama-kv-cache.h

```diff
@@ -15,7 +15,7 @@ struct llama_kv_cell {
 
     std::set<llama_seq_id> seq_id;
 
-    bool has_seq_id(const llama_seq_id & id) const {
+    bool has_seq_id(const llama_seq_id id) const {
```

Comment:
Remove this change

Reply:
fix it https://github.com/ggerganov/llama.cpp/pull/11294/commits/44ec40a43a3b6302a50aa934d8148cb8f5c11b71